card/src/component/Card.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import style from "../Card.module.css";
+import Cardcomp from "../Cardcomp";
+
+class Card extends Component {
+  componentDidMount() {
+    this.props && this.props.getCard();
+  }
+  render() {
+    return (
+      <div className={style.api}>
+        {this.props.userData &&
+          this.props.userData.userData &&
+          this.props.userData.userData.map(usr => {
+            return (
+              <Cardcomp
+                src={usr.avatar_url}
+                login={usr.login}
+                name={this.props.name}
```

Comment:
remove this because your using name in login

Reply:
done